docs/docs-content/clusters/cluster-groups/create-cluster-group.md

```diff
@@ -53,6 +53,12 @@ s
 
 :::
 
+:::info
+
+ Palette do not offer support for host clusters of the following types within the cluster group: edge clusters, virtual clusters, pcg clusters, and read-only imported clusters.
```

Comment:
```suggestion
 Palette does not offer support for host clusters of the following types within a cluster group: edge clusters, virtual clusters, Private Cloud Gateway (PCG) clusters, and imported clusters that have read-only access.
```

Reply:
In case you wonder why I suggested "these" vs. "the following" - it's your call, but I think "the following" lends itself better to a bullet list that follows.